tools/osv-scanner/cmd/osv-scanner/main.go

```diff
@@ -10,56 +10,73 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/BurntSushi/toml"
+	"github.com/g-rath/osv-detector/pkg/lockfile"
 	"github.com/google/osv.dev/tools/osv-scanner/internal/osv"
 	"github.com/google/osv.dev/tools/osv-scanner/internal/output"
 	"github.com/google/osv.dev/tools/osv-scanner/internal/sbom"
-
-	"github.com/g-rath/osv-detector/pkg/lockfile"
 	"github.com/urfave/cli/v2"
+	"golang.org/x/exp/slices"
 )
 
+const osvScannerConfigName = "osv-scanner.toml"
+
+var globalConfig *Config
```

Comment:
Maybe rename this to `configOverride` or something similar? It wasn't clear to me what exactly this meant at first read, and I thought this referred to a config that exists in say `$HOME/.osv.config` or something similar rather than an explicit config someone passed in.

See below though, for a possible refactor to make all of this cleaner and avoid this global var completely.